### PR TITLE
enforces multiline assignment style

### LINKED ISSUE #144: Enforce MultilineAssignmentLayout cop
> @tank-bohr it's better to format like this:

```
source =
  if ...
    ...
  else
    ...
  end
```

_Originally posted by @yegor256 in https://github.com/zerocracy/baza/issues/143#issuecomment-2254363333_

### PR DESCRIPTION
**Pull request by tank-bohr**
fixes #144

### CODE REVIEW

**Comment by yegor256** [comment]
@tank-bohr don't forget to [ping me](https://www.yegor256.com/2024/04/01/ping-me-please.html), otherwise I simply miss your pull requests

File: .rubocop.yml [hunk 1]
```diff
@@ -67,3 +67,5 @@ Metrics/AbcSize:
   Enabled: false
 Metrics/ParameterLists:
   Enabled: false
+Layout/MultilineAssignmentLayout:
+  Enabled: true
```
Comment (yegor256) [inline .rubocop.yml:71]: @tank-bohr I wonder, why it's not enabled by default? As far as I understand, Rubocop enforces _all_ rules by default, unless we disable them in the config file. If this is not true, then we have a bigger problem than this particular one.

**Review by yegor256 — CHANGES_REQUESTED** [review]
@tank-bohr looks perfect, but check my comment above. Maybe we should fix Rubocop config somehow?

File: .rubocop.yml (hunk 1 above)
Comment (tank-bohr) [reply .rubocop.yml:71]: [Docs](https://docs.rubocop.org/rubocop/configuration.html#enabled) say:

> Most cops are enabled by default. Cops, introduced or significantly updated between major versions, are in a special pending status (read more in ["Versioning"](https://docs.rubocop.org/rubocop/versioning.html)). Some cops, configured the above Enabled: false in [config/default.yml](https://github.com/rubocop/rubocop/blob/master/config/default.yml), are disabled by default.


изображение


To change the defaults as far as I understand you need to

```yaml
AllCops:
  EnabledByDefault: true
```

**Comment by yegor256** [comment]
@tank-bohr let's add this maybe:

```
AllCops:
  EnabledByDefault: true
```

**Pull request closed without merging**